src/axom/slic/interface/c_fortran/wrapSLIC.h

```diff
@@ -26,11 +26,11 @@ extern "C" {
 //  Level
 enum SLIC_Level
 {
-  Error,
-  Warning,
-  Info,
-  Debug,
-  Num_Levels
+  SLIC_Level_Error,
+  SLIC_Level_Warning,
+  SLIC_Level_Info,
+  SLIC_Level_Debug,
+  SLIC_Level_Num_Levels
```

Comment:
@ltaylor16 could you explain how a Fortran code would access these enums? Would it be possible to name them `SLIC_Error` instead of `SLIC_Level_Error` or

Reply:
I changed the names to remove Level.  I'm still working on Shroud changes which need to include namespace and class names into the member names.  This is to deal with the case where the same enum name is used in several scoped enums. C needs to flatten the scopes. This change should get the user going without having to update TPLs as part of the update.

Fortran generates `integer(C_INT), parameter :: level_error = 0`.  Like C, the `level` here is probably redundant since it's not possible to have another symbol at the same level named `error`.  Notice that it does not have a prefix like C does.  Since this is in a module, the user can resolve conflicts by controlling which symbol is used.  For example, `use slic_mod, only : level_error`.

Reply:
Great -- thanks for the explanation.